### PR TITLE
sig-docs: update member list

### PR DESCRIPTION
**Pull request by ran-huang**
Signed-off-by: Ran <huangran@pingcap.com>

Update Docs SIG's member list.

*We have 5 new contributors in the past 2 weeks!*

### CODE REVIEW

File: special-interest-groups/sig-docs/member-list.md [hunk 1]
```diff
@@ -34,13 +34,18 @@
...
 * [3pointer](https://github.com/3pointer)
+* [amyangfei](https://github.com/amyangfei)
 * [anotherrachel](https://github.com/anotherrachel)
 * [aylei](https://github.com/aylei)
 * [breeswish](https://github.com/breeswish)
 * [crazycs520](https://github.com/crazycs520)
 * [ericsyh](https://github.com/ericsyh)
+* [GMHDBJD](https://github.com/GMHDBJD)
+* [ireneontheway](https://github.com/ireneontheway)
+* [Joyinqin](https://github.com/Joyinqin)
```
Comment (lilin90) [inline special-interest-groups/sig-docs/member-list.md:45]: @yikeke What do you think?

Comment (lilin90) [inline special-interest-groups/sig-docs/member-list.md:45]:
```suggestion
```

Comment (ran-huang) [reply special-interest-groups/sig-docs/member-list.md:45]: Got it.

**Review by yikeke — APPROVED** [review]
LGTM

**Review by lilin90 — APPROVED** [review]
LGTM

**Comment by lilin90** [comment]
@winkyao PTAL

**Review by winkyao — APPROVED** [review]

**Comment by ran-huang** [comment]
Congratulations to @amyangfei, @GMHDBJD, @lance6716, @zimulala, you are now Reviewers of Docs SIG! 

Congratulations to @lidezhu, you are now an Active Contributor of Docs SIG! 

Congratulations to @2014BDuck, @EspenDai, @time-and-fate, @XiaoGe2030, @xxj123go, you are now Contributors of Docs SIG!

Thank you all for your contributions to TiDB documentation.

For more information on the roles and organization management of Docs SIG, see [roles-and-organization-management](https://github.com/pingcap/community/blob/master/special-interest-groups/sig-docs/roles-and-organization-management.md). And welcome to [join the Slack channel of Docs SIG](https://slack.tidb.io/invite?team=tidb-community&channel=sig-docs&ref=pingcap-community) for more communication!

**Pull request merged**